### PR TITLE
Clean-up white-space in some of the source files

### PR DESCRIPTION
**Pull request by Nirmal4G**
Part of #4779

### Context
Clean-up white-space in some of the source files (_files that could be a part of my later PRs_).

### Changes Made
1. Remove all trailing spaces.
2. Fixup new-line(s) where necessary
    - Add New Line(s) between every block to make it clear.
    - Remove unnecessary New Line(s) to reduce scrolling.
3. Remove unnecessary file (`src/Framework/Event args classes.cd`)

### Notes
These are the files that would be touched by later PRs.
Separating the white-space changes early on would help reviewers.

### CODE REVIEW

File: src/Shared/UnitTests/FileMatcher_Tests.cs [hunk 1]
```diff
@@ -2112,7 +2112,7 @@ private void GetMatchingDirectories(string[] candidates, string path, string pat
                             int nextSlash = normalizedCandidate.IndexOfAny(FileMatcher.directorySeparatorCharacters, path.Length + 1);
                             if (nextSlash != -1)
                             {
-                                
+
```
Comment (KirillOsenkov) [inline src/Shared/UnitTests/FileMatcher_Tests.cs:2115]: there shouldn't be an empty line here at all

File: src/Deprecated/Engine/Shared/FrameworkLocationHelper.cs [hunk 2]
```diff
@@ -66,8 +66,8 @@ internal static class FrameworkLocationHelper
         internal const string secondaryDotNetFrameworkSdkInstallKeyValueV40 = "CurrentInstallFolder";
         private const string dotNetFrameworkRegistryKeyV40 = dotNetFrameworkSetupRegistryPath + "\\v4\\Full";
         private static readonly GetDirectories getDirectories = new GetDirectories(Directory.GetDirectories);
-        
-        
+
+
```
Comment (KirillOsenkov) [inline src/Deprecated/Engine/Shared/FrameworkLocationHelper.cs:70]: never two consecutive empty lines in .cs files, just remove one of them

**Review by KirillOsenkov — APPROVED** [review]

File: eng/common/build.ps1 [hunk 3]
```diff
@@ -32,7 +32,7 @@ Param(
 # Unset 'Platform' environment variable to avoid unwanted collision in InstallDotNetCore.targets file
 # some computer has this env var defined (e.g. Some HP)
 if($env:Platform) {
-  $env:Platform=""
```
Comment (rainersigwald) [inline eng/common/build.ps1:35]: This file is not really part of our repo; it's copied here from https://github.com/dotnet/arcade/blob/main/eng/common/build.ps1. Please revert the change (it will just get overwritten on the next Arcade update).

File: src/Deprecated/Engine/Shared/FrameworkLocationHelper.cs (hunk 2 above)
Comment (rainersigwald) [reply src/Deprecated/Engine/Shared/FrameworkLocationHelper.cs:70]: agreed

```suggestion
```

File: src/Shared/UnitTests/FileMatcher_Tests.cs (hunk 1 above)
Comment (rainersigwald) [reply src/Shared/UnitTests/FileMatcher_Tests.cs:2115]:
```suggestion
```

**Review by rainersigwald — COMMENTED** [review]
Please don't rename `src/Framework/Event args classes.cd`. We should probably just delete it (in a separate PR).

Looking good overall!

**Comment by Nirmal4G** [comment]
`src/Framework/Event args classes.cd` is just one file. Do we need a separate PR for that? Are there any other files that are not needed anymore?

File: scripts/Enumerate-MSBuild.ps1 [hunk 4]
```diff
@@ -33,7 +33,7 @@ foreach ($instance in $vsInstances)
     $instancePath = $instance.installationPath
     Write-Log "********************" -LogToConsole $False
     Write-Log "Found VS Instance: $instanceName"
-
```
Comment (rainersigwald) [inline scripts/Enumerate-MSBuild.ps1:36]: Just realized this file was renamed. Please don't do that. Your name is better but we have links all over the Internet pointing to the old bad name.

**Review by rainersigwald — CHANGES_REQUESTED** [review]
You can leave the deletion of the `.cd` file in this PR if you rebase it into its own commit, please.

File: scripts/Enumerate-MSBuild.ps1 (hunk 4 above)
Comment (Nirmal4G) [reply scripts/Enumerate-MSBuild.ps1:36]: I see! Didn't think of it as an intrusive change!! Sorry!!!

**Comment by rainersigwald** [comment]
Looks good! I don't think this meets the bar for 16.11, so I retargeted to `main`. Which naturally caused some merge conflicts because #6547 hasn't landed yet. But "fortunately" we can't check in to main right now because of problems on the Visual Studio side. I don't think you'll need to do anything, and we should be good to go in a few days. Thanks!

(leaving "change requested" so I remember to check on this but I actually approve)

**Comment by KirillOsenkov** [comment]
So, could you say that this PR... nirmalizes whitespace?

**Comment by Nirmal4G** [comment]
@KirillOsenkov There's lot more to nirmalize!! 😜😂

**Review by rainersigwald — APPROVED** [review]
Looks unblocked to me. Not sure why GitHub shows the diff as including release notes and stuff; those commits don't seem to be different. I'm going to close/reopen the PR and see if that helps it.

**Review by Forgind — APPROVED** [review]
The extra commits you noticed are still there, but that doesn't bother me too much. Fine with me if rainersigwald is fine with it.

Thanks Nirmal4G!

**Comment by rainersigwald** [comment]
They weren't really there; I manually merged to be extra sure of that :)

Thanks, @Nirmal4G!
